botsing-maven/pom.xml

```diff
@@ -67,6 +67,12 @@
 			<artifactId>maven-aether-provider</artifactId>
 			<version>${maven-aether-provider.version}</version>
 		</dependency>
+		
+		<dependency>
+		    <groupId>org.apache.maven.plugins</groupId>
+		    <artifactId>maven-dependency-plugin</artifactId>
+		    <version>3.1.1</version>
```

Comment:
The version is hardcoded and should be specified in the pom.xml file at the root level of the project.

Reply:
Moved to parent pom